qemu/target-mips/cpu.c

```diff
@@ -151,15 +151,17 @@ void mips_cpu_register_types(void *opaque)
         
         sizeof(MIPSCPUClass),
         sizeof(MIPSCPU),
-		opaque,
+        opaque,
         
-		mips_cpu_initfn,
-		NULL,
-		NULL,
+        mips_cpu_initfn,
+        NULL,
+        NULL,
+
+        NULL,
```

Comment:
ah it seems it broke before because you missed a NULL above? this is why we should avoid doing this struct initialization when possible (which is avoidable here)